styles/styles.css

```
@@ -966,13 +977,29 @@ main > .section > .grid-container {
 }
 
 .theme--hero-green {
-  --page-bg-overscroll-color: var(--spectrum-green-300);
-  --page-bg-main-gradient: var(--gradient-green);
+  --page-gradient-start-color: var(--gradient-hero-green-start);
   --diamond-fill: var(--spectrum-green-600);
 }
 
-.theme--hero-indigo .bg-visuals .shape-diamond,
-.theme--hero-green .bg-visuals .shape-diamond {
+.theme--hero-indigo,
+.theme--hero-green {
+  --page-bg-height: 48.813rem;
+  
+  @media (min-width: 48rem) {
+    --page-bg-height: 70.75rem;
+  }
+
+  @media (min-width: 80rem) {
+    --page-bg-height: 36.563rem;
+  }
+
+  @media (min-width: 105rem) {
+    --page-bg-height: 55.938rem;
+  }
+}
+
+.bg-visuals .shape-diamond,
```

```suggestion
.background-visuals .shape--diamond,
```

I'd like `shape` to be more BEM, I know this was on your other PR